### ISSUE TITLE
Not recognized modifier, Forbidden Shako Great Crown

### ISSUE THREAD

**Issue by Nevalopo** [SnosMe/awakened-poe-trade#1454]
I hope this is eventually fixed, It seems like such a minor thing to fix and has been like this since like 6 leagues back.

You are unable to price check Forbidden Shako Great Crown for some reason.

Socketed Gems are Supported by Level 6(1-10) Predator(Greater Multiple Projectiles-Expert Retaliation)
explicit — Not recognized modifier
Socketed Gems are Supported by Level 33(25-35) Prismatic Burst(Greater Multiple Projectiles-Expert Retaliation)
explicit — Not recognized modifier

**Comment by Sur-code** [comment]
Yeah, I hope the author will fix this, really it takes a lot of time for checking 😅
